Containers/talk/start.sh

```diff
@@ -16,43 +16,46 @@ elif [ -z "$INTERNAL_SECRET" ]; then
 fi
 
 set -x
-IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk A +short)"
+if [ -n "$(dig nextcloud-aio-talk A +short | grep -P "^[0-9.]+$" | head -n1)" ]; then
+    IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk A +short | grep -P "^[0-9.]+$" | head -n1)"
+fi
+if [ -n "$(dig nextcloud-aio-talk AAAA +short | grep -P "^[0-9a-fA-F:]+$" | head -n1)" ]; then
+    IPv6_ADDRESS_TALK="$(dig nextcloud-aio-talk AAAA +short | grep -P "^[0-9a-fA-F:]+$" | head -n1)"
+fi
```

Comment:
this should always only return one ip-adress or none at all, no?

Reply:
Yes, eturnal currently can handle only one IPv6 and/or IPv4 address to advertise for TURN relays.

Reply:
the ipv6 ip to talk container (or nothing if no ipv6)

Reply:
What I meant was this:
```suggestion
IPv4_ADDRESS_TALK="$(dig nextcloud-aio-talk AAAA +short)"
```

Reply:
but we need the ipv6 ip?

Reply:
Yes and we get it by so, no?

Reply:
yes but the other way checks, that is just one ip and that the ip is an ipv6 ip

Reply:
It will always be just one ip and a ipv6 ip, no? (I think the check is not necessary here)

Reply:
I would keep it

Reply:
Why? I dont understand how this command can retutn more than one value

Reply:
I know unrealistic, but multiple docker networks

Reply:
But the talk container is by design only added to the nextcloud-aio network. How should this possibly happen?

Reply:
the user connects the container to an additional network

Reply:
Ehy should they do this?

Reply:
there is no reason, but I would prefer to keep it like this, so that we always get a correct ip

Reply:
hm... honestly I would rather make this more simple if there is no reason...

Reply:
And I would keep it...